proto/dymensionxyz/dymension/sequencer/sequencer.proto

```diff
@@ -16,32 +16,42 @@ import "cosmos/msg/v1/msg.proto";
 // Sequencer defines a sequencer identified by its' address (sequencerAddress).
 // The sequencer could be attached to only one rollapp (rollappId).
 message Sequencer {
-  // sequencerAddress is the bech32-encoded address of the sequencer account which is the account that the message was sent from.
+  // sequencerAddress is the bech32-encoded address of the sequencer account
+  // which is the account that the message was sent from.
   string sequencerAddress = 1;
-  // pubkey is the public key of the sequencers' dymint client, as a Protobuf Any.
-  google.protobuf.Any dymintPubKey = 2 [(cosmos_proto.accepts_interface) = "cosmos.crypto.PubKey"];
+  // pubkey is the public key of the sequencers' dymint client, as a Protobuf
+  // Any.
+  google.protobuf.Any dymintPubKey = 2
+      [ (cosmos_proto.accepts_interface) = "cosmos.crypto.PubKey" ];
   // rollappId defines the rollapp to which the sequencer belongs.
   string rollappId = 3;
   // description defines the descriptive terms for the sequencer.
-  Description description = 4 [(gogoproto.nullable) = false];
+  Description description = 4 [ (gogoproto.nullable) = false ];
 
-  // jailed defined whether the sequencer has been jailed from bonded status or not.
+  // jailed defined whether the sequencer has been jailed from bonded status or
+  // not.
   bool jailed = 5;
 
-// proposer defines whether the sequencer is a proposer or not.
-   bool proposer = 6;
+  reserved 6;
 
   // status is the sequencer status (bonded/unbonding/unbonded).
   OperatingStatus status = 7;
 
   // tokens define the delegated tokens (incl. self-delegation).
-  repeated cosmos.base.v1beta1.Coin tokens = 8 
-  [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
-
-    // unbonding_height defines, if unbonding, the height at which this sequencer has begun unbonding.
-  int64 unbonding_height = 9;
-  // unbond_time defines, if unbonding, the min time for the sequencer to complete unbonding.
-  google.protobuf.Timestamp unbond_time = 10 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
-
+  repeated cosmos.base.v1beta1.Coin tokens = 8 [
+    (gogoproto.nullable) = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+
+  // unbond_request_height stores the height at which this sequencer has
+  // requested to unbond.
+  int64 unbond_request_height = 9;
+  // unbond_time defines, if unbonding, the min time for the sequencer to
+  // complete unbonding.
+  google.protobuf.Timestamp unbond_time = 10
+      [ (gogoproto.nullable) = false, (gogoproto.stdtime) = true ];
+
+  // notice_period_time defines the time when the sequencer will finish it's notice period if started
+  google.protobuf.Timestamp notice_period_time = 11
```

Comment:
not sure I understand why is this an attribute of the sequencer vs being calculated when needed?